packages/shortest/src/cli/bin.ts

```diff
@@ -111,6 +111,8 @@ function isValidArg(arg: string): boolean {
 async function main() {
   const args = process.argv.slice(2);
 
+  const legacyOutputEnabled = !args.includes("--no-legacy-output");
```

Comment:
Temporary flag used for https://github.com/anti-work/shortest/pull/320 to test before/after. It will be removed before the next release.